spec/sections/copies.bs

```diff
@@ -29,8 +29,9 @@ and "immediate" {{GPUQueue}} operations:
 - {{GPUQueue/copyExternalImageToTexture()}}, for copies from Web Platform image sources to textures
 
 Some texel values have <dfn dfn lt='equivalent texel representation'>multiple possible representations</dfn>
-of some values, e.g. as `r8snorm`, -1.0 can be represented as either -127 or -128.
-Copy commands are not guaranteed to preserve the source's bit-representation.
+of some values, e.g. as `r8snorm`, -1.0 can be represented as either -127 or -128. Additionally NaN,
+zero, and subnormal float values may also have multiple equivalent representations. Copy commands
+are not guaranteed to preserve the source's bit-representation.
```

Comment:
Looking at this written out, I think we actually might need to change the concepts here so it's more broad - if we assume a copy can go through WGSL, and WGSL NaN produces indeterminate values, then a copy can too.

Maybe we can reference WGSL about this: https://gpuweb.github.io/gpuweb/wgsl/#differences-from-ieee754 (It also mentions infinities which I forgot about.)

IIUC:

- snorm `-127` &harr; `-128`
- any NaN or infinity &rarr; indeterminate value (but not signaling NaN)
- any subnormal or `-0.0` or `+0.0` &rarr; `-0.0` or `+0.0`

Reply:
Updated the language to more closely echo what you listed here, as well as adding a non-normative note that WGSL may be used for copies and so any of the WGSL float behaviors may be observed.